modules/ROOT/pages/deployment/container/container-setup.adoc

```diff
@@ -158,6 +177,73 @@ sudo docker rmi <image-id>
 
 == Start the oCIS Runtime
 
+The start of oCIS is made in two steps:
+
+* A first time start to initiate the system and
+* A recurring start post initialisation.
+
+Note the xref:deployment/general/general-info.adoc#default-users-and-groups[Default Users and Groups] section if you want to have demo users created when inizializing the system.
+
+=== Preperation
+
+In the examples shown, bind mounts with following folders are used to keep data persistent, located in your home directory. Change the locations according your needs:
+ 
+* For the config the folder `~/ocis/ocis-config`
+* For the data the folder `~/ocis/ocis-data`
+
+The oCIS container runs internally with the default user and group ID of 1000. To check this type:
+
+[source,bash]
+----
+sudo docker inspect owncloud/ocis | less | grep -i user | sort -u
+----
+[source,plaintext]
+----
+"User": "1000",
+----
+
+For the folders above, following rules apply:
+
+* Because _bind-mounts_ used below create paths if they do not extist with the root user and group, the container user can not write into them. To overcome this issue, you have to create both folders upfront manually to avoid a permission denied problem. To do so type:
++
+--
+[source,bash]
+----
+mkdir -p ~/ocis/ocis-config
+----
+[source,bash]
+----
+mkdir -p ~/ocis/ocis-data
+----
+--
+
+* The user you have logged in with must have the default user and group ID of 1000 to match the container user and group ID. If this does not match, you must take care that the container user can write into the created paths. As an example to do so type:
++
+--
+[source,bash]
+----
+sudo chown -Rfv 1000:1000 ocis/
+----
+In case, you now need to access the content of the `ocis` folder with root previledges or with a user matching the ID's.
+--
+
+=== First Time Start
+
+oCIS needs a first time initialisation which will setup your enviroment. You will get questions asked and a default `ocis.yaml` file will be generated which you further can configure. Note that if you do not define a host directory like with a bind mount as target location, the initial setup will get lost because the container ends post executing the command. ownCloud therefore recommends either using bind mounts or docker volumes to make the initial setup, changes and your data persistent.
```

Comment:
```suggestion
oCIS needs a first time initialization to set up the environment. You will need to answer questions as basis for a default `ocis.yaml` file to be generated. This file you can edit later. Note that if you do not define a host directory with a bind mount as target location, the initial setup will get lost because the container ends after executing the command. ownCloud therefore recommends either using bind mounts or Docker volumes to make the initial setup, further changes and your data persistent.
```